src/app/app.component.ts

```diff
@@ -208,6 +209,99 @@ export class AppComponent implements OnInit {
       this.schema,
       this.dataSources.dataSources
     );
+
+    // Get concepts with no label
+    const concepts = this.traverseForUnlabeledConcepts(this.form.rootNode);
```

Comment:
Here we'll get all the concepts from the JSON schema that have no label specified